consensus/fork_choice/src/fork_choice.rs

```diff
@@ -127,6 +127,17 @@ pub enum PayloadVerificationStatus {
     Irrelevant,
 }
 
+impl PayloadVerificationStatus {
+    /// Returns `true` if the payload was optimistically imported.
+    pub fn is_optimistic(&self) -> bool {
```

Comment:
I'm wondering if `PayloadVerificationStatus::NotVerified` should be renamed to `PayloadVerificationStatus::Optimistic` as well to make this mapping very clear. I could understand if we want to keep the concept of "optimism" constrained to fork choice though. What are your thoughts?

This method can also be used in `block_verification` L1168 by the way

Reply:
Great suggestions, will do both.